core/src/main/java/org/apache/druid/segment/column/ValueType.java

```diff
@@ -63,23 +63,37 @@ public enum ValueType implements TypeDescriptor
    * String object type. This type may be used as a grouping key, an input to certain types of complex sketch
    * aggregators, and as an input to expression virtual columns. String types might potentially be 'multi-valued' when
    * stored in segments, and contextually at various layers of query processing, but this information is not available
-   * through this enum alone, and must be accompany this type indicator to properly handle.
+   * at this level.
+   *
+   * Strings are typically represented as {@link String}, but multi-value strings might also appear as a
```

Comment:
"Might"?

How about: "multi-value strings appear as {@code java.util.List<String>} when necessary to represent multiple values."

Reply:
I guess i intended to convey that it can vary from value to value/row to row, but on second reading agree its not entirely clear, will make some adjustments